apostrophecms/apostrophe
# update CKEDITOR

## Summary

* Update CKEDITOR to 4.22.1
* Add Font plugin

## What are the specific steps to test this change?

1. Update `sanitizeHtml` options with the following
   ```js
   // lib/modules/apostrophe-rich-text-widgets/index.js
   module.exports = {
     sanitizeHtml: {
       allowedAttributes: {
         a: [ 'href', 'name', 'target' ],
         img: [ 'src', 'srcset', 'alt', 'title', 'width', 'height', 'loading' ],
         span: [ 'style' ]
       }
     }
   };
   ```
2. Add `Font` and `FontSize` to the toolbar
   ```js
   toolbar = [ 'Styles', 'Bold', 'Italic', 'Blockquote', 'Link', 'Anchor', 'Unlink', 'NumberedList', 'BulletedList', 'Split', 'Font', 'FontSize' ];
   ```
3. Edit a rich-text content (adding links, anchors, styles, font and font size, ...). When you save the data and reload the page, the content and style should remain the same.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
